app/Filament/Resources/AchievementResource.php

```diff
@@ -454,6 +517,90 @@ public static function getEloquentQuery(): Builder
             ->withoutGlobalScopes([
                 SoftDeletingScope::class,
             ])
-            ->with(['game']);
+            ->with(['activeMaintainer.user', 'game']);
+    }
+
+    public static function buildMaintainerForm(): array
+    {
+        return [
+            Forms\Components\Placeholder::make('ticket_info')
+                ->hiddenLabel()
+                ->content('The new maintainer will inherit any open tickets for this achievement.')
+                ->extraAttributes(['style' => 'color: oklch(82.8% 0.189 84.429)']), // amber-400 (https://tailwindcss.com/docs/colors#color-palette-reference)
+
+            Forms\Components\Select::make('user_id')
+                ->label('Maintainer')
+                ->searchable()
+                ->getSearchResultsUsing(function (string $search): array {
+                    return User::query()
+                        ->whereHas('roles', function ($query) {
+                            $query->where('name', Role::DEVELOPER);
+                        })
+                        ->where('display_name', 'LIKE', "%{$search}%")
+                        ->limit(50)
+                        ->pluck('display_name', 'id')
+                        ->toArray();
+                })
```

Comment:
I'm not sure how important it is, but you can't assign back to the original author if they're no longer active (registered/retired) or a junior developer.

Reply:
Fixed in latest. You can now always set the maintainer to be the achievement's original author.

Reply:
This creates a weird juxtaposition where the original author appears in the list even when not searching for them.
![image](https://github.com/user-attachments/assets/6905afff-59b0-4a6b-b670-a4c0c9139d50)

Reply:
Resolved in latest.